tests/structs/test_smtp_account.py

```diff
@@ -1,84 +1,85 @@
-import unittest
-
 import pytest
-
 from structs.email.smtp_account import SMTPAccount
 
 
-class TestEmailParams(unittest.TestCase):
-    """
-    Tests that methods in EmailParams dataclass work expectedly
-    """
-
-    def setUp(self) -> None:
-        """setup for tests"""
-        self.mock_smtp_accounts_config = [
-            {
-                "name": "config1",
-                "username": "user1",
-                "password": "some-pass",
-                "server": "sever",
-                "port": "port",
-                "secure": True,
-                "smtp_auth": True,
-            },
-            {
-                "name": "config2",
-                "username": "user2",
-                "password": "some-pass2",
-                "server": "sever",
-                "port": "port",
-                "secure": False,
-                "smtp_auth": False,
-            },
-        ]
-        self.mock_pack_config = {"smtp_accounts": self.mock_smtp_accounts_config}
-
-    def test_from_dict(self):
-        """
-        Tests that from_dict() static method works properly
-        this method should build a SMTPAccount dataclass from a valid dictionary
-        """
-
-        mock_valid_kwargs = {
+@pytest.fixture(name="mock_smtp_accounts")
+def mock_smtp_accounts_fixture():
+    return [
```

Comment:
Docstring - this should mention that it includes multiple smtp accounts as a dict

We should also add a type annotation here too as we have a data struct with a similar name